src/FeatureConstraints.cpp

```
@@ -31,7 +31,7 @@ namespace FeatureConstraints
 							feature->GetName(), FormatConstraintValue(constraint.forcedValue),
 							result.sources[0].featureName, FormatConstraintValue(result.forcedValue));
 					}
-					result.sources.push_back({ feature->GetName(),
+					result.sources.push_back({ feature->GetDisplayName(),
 						feature->GetShortName(),
 						constraint.reason,
```

Cppcheck (2.21.0)

[error] Returning pointer to local variable 'fields' that will be invalid when returning.

(returnDanglingLifetime)